src/main/java/com/smiley/smileybackend/controller/MedicineController.java

```diff
@@ -0,0 +1,35 @@
+package com.smiley.smileybackend.controller;
+
+import com.smiley.smileybackend.dto.response.MedicineInfoDto;
+import com.smiley.smileybackend.dto.user.MedicineIdDto;
+import com.smiley.smileybackend.service.MedicineService;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import lombok.ToString;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.validation.Valid;
+
+@ToString
+@RestController
+@Api(tags = "Medicine Controller :  의약품 정보")
+@Slf4j
+
+public class MedicineController {
+    private MedicineService medicineService;
+    public MedicineController(MedicineService medicineService){
+        this.medicineService = medicineService;
+    }
+
+    @PostMapping("/medicine")
```

Comment:
url 복수 형태로 바꿔주시고 Id만 있는 Dto를 만들어 PostMapping을 사용하는 것 대신 GetMapping으로 하는게 더 좋아보입니다.

```suggestion
@GetMapping("/medicines/{id}")
```